pom.xml

```diff
@@ -27,6 +29,33 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!--mybatis 起步依赖-->
+        <dependency>
+            <groupId>org.mybatis.spring.boot</groupId>
+            <artifactId>mybatis-spring-boot-starter</artifactId>
+            <version>3.0.2</version>
+        </dependency>
+
+        <!-- mysql驱动包 , 刚刚发布的最新版本的驱动包-->
```

Comment:
If you need comments, then comment with English

Reply:
ok